### PR TITLE
Noreject staff

### PR DESCRIPTION
**Pull request by kkamalov**

### CODE REVIEW

File: orchestra/communication/staffing.py [hunk 1]
```diff
@@ -21,20 +22,22 @@ def handle_staffing_response(worker, staffing_request_id, is_available=False):
...
     else:
         response = StaffingResponse.objects.create(
             request=staffing_request,
             is_available=is_available)
 
     if (is_available and
-            not StaffingResponse.objects.filter(is_winner=True).exists()):
+            not StaffingResponse.objects.filter(
+                    request__task=staffing_request.task,
+                    request__required_role=staffing_request.required_role,
```
Comment (marcua) [inline orchestra/communication/staffing.py]: how does this work for multiple levels of review?  do we need a required_role_level as well?

File: orchestra/templates/communication/staffing_response_not_permitted.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+{% block title %}
+Task cannot be rejected.
+{% endblock %}
+
+{% block main %}
+<div class="container-fluid main-content communication-container">
+  <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+    <div class="card">
+      Unfortunately, task cannot be rejected after it has been accepted.
```
Comment (marcua) [inline orchestra/templates/communication/staffing_response_not_permitted.html]: Unfortunately, you can not reject the task after accepting it.  Reach out if you can no longer work on this task!

File: orchestra/utils/task_lifecycle.py [hunk 3]
```diff
@@ -156,6 +158,16 @@ def remove_worker_from_task(worker_username, task_id):
     task_assignment.status = TaskAssignment.Status.FAILED
     task_assignment.save()
     task.save()
+
+    # if there is existing StaffingResponse mark is as not a winner
```
Comment (marcua) [inline orchestra/utils/task_lifecycle.py]: mark it

File: orchestra/communication/staffing.py (hunk 1 above)
Comment (kkamalov) [inline orchestra/communication/staffing.py]: yeah, that makes sense. let me address it in a separate PR.

**Comment by marcua** [comment]
lgtm --- i added a task to add multiple levels of review

File: orchestra/templates/communication/staffing_response_not_permitted.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+{% block title %}
+Task cannot be rejected.
+{% endblock %}
+
+{% block main %}
+<div class="container-fluid main-content communication-container">
+  <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+    <div class="card">
+      Unfortunately, you can not reject the task after accepting it. Reach out if you can no longer work on this task!
```
Comment (joshblum) [inline orchestra/templates/communication/staffing_response_not_permitted.html:12]: "Reach out in the project slack channel"

**Pull request merged**